CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.0.0-rc.1 (2019/10/08)
+
+- Added a custom build step to export `src/app/public/styles/ag-grid-styles.scss` and fixed minor styling issues. [#9](https://github.com/blackbaud/skyux-ag-grid/pull/9)
```

Comment:
Looks good to me. 

This one's probably just for my own knowledge, but do we need to call out the path for that stylesheet or can we just refer to it as the `ag-grid-styles.scss` and say "Added a custom build step to export `ag-grid-styles.scss`...?

Reply:
@Blackbaud-DeniseCisneros honestly I wasn't sure so I figured one of you would tell me in the PR, haha. Should I remove the path?